packages/nextra-theme-docs/src/contexts/config.tsx

```diff
@@ -58,21 +54,17 @@ export const ConfigProvider = ({
   }
   if (!isValidated) {
     try {
-      theme = themeSchema.parse(theme, {
-        errorMap: () => ({ message: 'Invalid theme config' })
-      })
```

Comment:
we should not hide the original error message, otherwise, for example if we set `defaultCollapseLevel: -1` we'll not understand what's wrong, now it's clearer
![image](https://user-images.githubusercontent.com/7361780/213897474-4dac5086-8ad0-4b9b-85c5-d0c521fb2d1d.png)

Reply:
I adjusted the path in https://github.com/shuding/nextra/pull/1343 PR by

```js
`Path: "${issue.path.join('.')}"`
```

like it was before, so the first error will end with `Path: "sidebar.defaultMenuCollapseLevel"`